### PR TITLE
Ported remaining PDF library examples

### PR DESCRIPTION
**Pull request by kingofools**
referencing issue: https://github.com/jdf/Processing.py-Bugs/issues/4

### CODE REVIEW

File: examples.py/Library/PDF Export/LargePage.pyde [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+def setup():
+    size(2000, 2000, PDF, "Line.pdf")
+
+def draw():
+    background(255)
+    stroke(0, 20)
+    strokeWeight(20.0)
+    line(200, 0, width/2, height)
+    
+    exit()
```
Comment (jdf) [inline examples.py/Library/PDF Export/LargePage.pyde]: Thanks for this.
Please auto-format each of the examples.

File: examples.py/Library/PDF Export/LargePage.pyde [hunk 2]
```diff
@@ -1,22 +0,0 @@
...
-def setup():
-    size(2000, 2000, PDF, "Line.pdf")
-
-def draw():
-    background(255)
-    stroke(0, 20)
-    strokeWeight(20.0)
-    line(200, 0, width/2, height)
-    
-    exit()
```
Comment (kingofools) [inline examples.py/Library/PDF Export/LargePage.pyde]: This was a mistake. I didn't intend to remove this.

File: examples.py/Library/PDF Export/LargePage.pyde [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+def setup():
+    size(2000, 2000, PDF, "Line.pdf")
+
+
+def draw():
+    background(255)
+    stroke(0, 20)
+    strokeWeight(20.0)
+    line(200, 0, width / 2, height)
+    exit()
```
Comment (kingofools) [inline examples.py/Library/PDF Export/LargePage.pyde]: fixing the wrong commit : https://github.com/jdf/processing.py/pull/249/commits/551e98bd23026f8dafe531b5cfe97e54135093ac

**Comment by jdf** [comment]
Thank you.

**Pull request merged**